### ISSUE TITLE
No querys work using the given examples 

### ISSUE THREAD

**Issue by Danjaraka** [furkanmtorun/gnomad_python_api#5]
Hi, 
Just found your API looks good so far 👍. Not sure if Im doing something wrong, but when using examples given no outputs are generated using  the CLI or even through the GUI. Even when using the default GUI parameters plots or data is not able to be generated.
![image](https://user-images.githubusercontent.com/38589122/101712963-a07c2b00-3ad1-11eb-828d-8390192539b0.png)

**Comment by furkanmtorun** [comment]
Dear Daniel (@Danjaraka ),

Firstly, thank you very much for taking your time to open this issue.

Yes, I have checked the API and realized that [gnomAD GraphQL API](https://gnomad.broadinstitute.org/api) was updated, and some query syntax and keywords were deprecated or altered. 
Hence, the batch script is currently not able to retrieve the data from the gnomAD API and not able to generate outputs and plots.

I hope I can find time to incorporate with the new updates!
Furthermore, any coding contribution or pull requests are also welcome!
 
Best,
Furkan

**Comment by S-Maxey90** [comment]
Hi both,
I've also encountered this problem today, I thought it might be helpful to note that for me, there is one functional example.
The search by RSID feature does still work 
`python gnomad_api_cli.py -filter_by=rs_id -search_by="rs201857604" -dataset="gnomad_r2_1"`
I don't know if that's any use to you @Danjaraka, but I thought I would be worth flagging!

**Comment by furkanmtorun** [comment]
Hi @S-Maxey90,
Thanks for your message!

Yes, most probably, the query syntax remained the same for searching with RS_ID.
For other searching options, some query syntax and keywords were deprecated or altered.

In this respect, any coding contribution or pull requests (PR) will be appreciated!

Best,
Furkan

**Comment by gorostiolam** [comment]
Hi all, 
I encountered the same problems and, after reading this thread, I looked a bit more into it. 

At the moment, I have a working script for retrieval of variants using "gene_symbol" as input (former "gene_name"), that I can share if you are interested. 

I discovered that one of the main problems with the current GraphQL API is a new response status, with code 400, that gives an error if the query is too expensive (> 25).  If you are still interested in retrieving all the data that was initially included in the script, you will probably need to loop over the different data types one at a time. In my script I did not do it because I was just interested in the variants, and that is a small enough query. Apart from that, as it was already mentioned, many keywords were deprecated/altered, but using the [gnomAD GraphQL API](https://gnomad.broadinstitute.org/api/) testing tool it was not too difficult to find the updated ones.  

In any case, @furkanmtorun kudos for a very nice tool! I was a bit worried that I would need to downlaod all the raw VCF files from gnomAD (~1TB) to look at the variants of the ~400 genes that I was interested in, and your tool completely saved me from doing that. It is a pity that they introduced all these changes, but luckily it is an easy fix.
Let me know if you are interested in me sharing the patch script. 

Best,

Marina

**Comment by furkanmtorun** [comment]
Dear Marina (@gorostiolam ),

First of all, I apologize for my late response and thanks for your contribution. 
Also, it is nice to hear that you like the tool!

Yes, updated or new keywords can be replaced. In addition to this, query syntaxes should be corrected since that part is also altered.

I would be very happy to see your patch script! 
Since I am currently not able to do the required changes, it would be so great to add your updated script to the repository!

Thank you very much for your all efforts in advance. 

Best,
Furkan

**Comment by Danjaraka** [comment]
Yes, please share your patch script. I would be happy to implement the changes into the API :)

**Comment by gorostiolam** [comment]
Dear Furkan,

Thanks for your message. ☺
I did not make it a pull request because I did not have the time to create the test environment, etc. But I would be happy to share the code with you if that is useful for other people as well.
You can find my script attached to this email.

Cheers,

Marina

**Comment by furkanmtorun** [comment]
Dear Marina (@gorostiolam  ),

Thank you very much for your support! I think we can add your script file to the repository now.

It seems that Daniel (@Danjaraka ) may help us in implementing the changes to the API.

However, unfortunately, while replying via email, it seems the files attached to emails are not included to the GitHub comments. 
Hence, I am not able to see your script file.

After you post or deliver the file, I would be very happy to add it to the repository and give your credits to the README file 👍 

Yours sincerely,
Furkan

**Comment by jpinero** [comment]
Hi! great script, and rather easy to adapt to make it work, even for non-python people :1st_place_medal: 
Quick question: does anyone know if it is possible to retrieve AF stratified by sex? 
Thanks anyway
janet

**Comment by furkanmtorun** [comment]
Hi Janet,

Thank you very much for your great comment!

Best,
Furkan

**Comment by xingjun29** [comment]
I hadn't seen update for the patch, still not working
